live/global.ini

```diff
@@ -65817,16 +65817,16 @@ spacecargo_illegal_obj_marker_02=Die Ware
 spacecargo_illegal_obj_short_01=Begib dich zur Position des Wracks
 spacecargo_illegal_obj_short_02=Schnapp dir die Ware
 spacecargo_illegal_obj_short_03=Liefere Waren
-spacecargo_illegal_threat_0001=Malen Sie die Innenseite Ihres Helms rot
-spacecargo_illegal_threat_0002=Sie wünschen sich, Sie wären auf Wega gewesen, als die Duul angegriffen haben
-spacecargo_illegal_threat_0003=Veranstalte einen Urlaub für dich im nächstgelegenen Stern
-spacecargo_illegal_threat_0004=Machen Sie ein Exempel an Ihnen
-spacecargo_illegal_timed_0001=Dafür gibt es eine Frist, denn wenn wir diesen wertvollen Gegenstand nicht schnell besorgen, wird jemand sterben. 
-spacecargo_illegal_timed_0002=Das muss schnell erledigt werden, also machen Sie sich keine Sorgen. 
-spacecargo_illegal_timed_0003=Erwarten Sie, dass Sie so schnell wie möglich dort ankommen, bevor irgendein Hopper über die ~mission(Ship) stolpert und es zuerst rettet. 
-spacecargo_illegal_timed_0004=Die ~mission(Client) braucht das schnell, also stelle sicher, dass du nicht zu spät kommst. 
-spacecargo_illegal_timed_0005=Dieses Zeug wird nicht für immer da draußen schweben, also kümmere dich darum, bevor es zu spät ist. 
-spacecargo_illegal_timed_0006=Die Möglichkeiten für diesen Job sind begrenzt. Passen Sie auf, dass Sie es nicht verpassen. 
+spacecargo_illegal_threat_0001=streiche den Helm von innen rot.
```

Comment:
Alle threats an den Satz oben anpassen

Reply:
Schau bitte in die engl. Datei. Der Kauderwelch steht dort genau so drin ^^

Reply:
Glaube ich dir, im englischen ist es aber angenehmer/leichter. Du musst Zeile spacecargo_illegal_desc_0005 mit den Threats paaren: 

`Eine Constellation mit wertvoller Fracht ist explodiert und der Frank zahlt dafür, dass die Sendung von Meth zur Grimhex geliefert wird. [...] Wenn ich höre, dass dieses Paket woanders landet und ich herausfinde, dass du etwas damit zu tun hast, dann werde ich "streiche den Helm von innen rot" ausführen.`

Also wäre für diese Zeile: deinen Helm von innen rot färben.
und für die "Elternzeile" das ausführen weg.

Reply:
Und natürlich müssen wir vlt auch andere Elternzeilen in Erwägung ziehen.

Von mir aus können wir uns auch mal die Zeilen raussuchen (in Issue) und zusammen grübeln. Wir hatten da ja auch schon welche an anderen Stellen.

Reply:
Ich hab es mal notiert und verschaffe mir nachher mal einen Überblick